pkg/gameserverallocations/controller_test.go

```diff
@@ -893,7 +893,7 @@ func TestMultiClusterAllocationFromRemote(t *testing.T) {
 				return true, getTestSecret(secretName, server.TLS.Certificates[0].Certificate[0]), nil
 			})
 
-		stop, cancel := agtesting.StartInformers(m)
+		stop, cancel := agtesting.StartInformers(m, c.secretSynced)
```

Comment:
agtesting.StartInformers(m, c.allocationPolicySynced, c.gameServerSynced, c.secretSynced) here and in other places?

Reply:
Nicely spotted! Yep - exactly. Pushing up a change now. :100: